### PR TITLE
Use isohybrid only in amd64 architecture (#1831)

### PR DESCRIPTION
**Pull request by kalxas**

### CODE REVIEW

**Comment by bencaradocdavies** [comment]
@kalxas ishybrid will work on i386; this change removes both isohybrid and UEFI, when all that is needed is to remove UEFI.

File: bin/build_chroot.sh [hunk 1]
```diff
@@ -310,12 +310,19 @@ echo
...
+      isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot \
+      -boot-load-size 4 -boot-info-table \
+      -eltorito-alt-boot -e boot/grub/efi.img -no-emul-boot \
+      -o ../"$ISO_NAME.iso" .
+   sudo isohybrid -u ../"$ISO_NAME.iso"
+else
+   sudo mkisofs -D -r -V "$IMAGE_NAME" -cache-inodes -J -l -quiet -b \
+      isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot \
+      -boot-load-size 4 -boot-info-table -o ../"$ISO_NAME.iso" .
+fi
```
Comment (bencaradocdavies) [inline bin/build_chroot.sh]: You can insert ```isohybrid ../"$ISO_NAME.iso"``` at the end of the non-amd64 block to make the ISO a non-UEFI isohybrid (no ```-u``` flag).

Comment (kalxas) [inline bin/build_chroot.sh]: oops, you are right, will fix

**Comment by bencaradocdavies** [comment]
@kalxas that looks good to me.

For the record, there is such a thing as i386 UEFI, it is just that Ubuntu does not support UEFI on i386.

File: bin/build_chroot.sh [hunk 2]
```diff
@@ -310,12 +310,20 @@ echo
 echo "Creating iso..."
 echo "======================================"
 #Create the ISO image
-sudo mkisofs -D -r -V "$IMAGE_NAME" -cache-inodes -J -l -quiet -b \
-   isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot \
-   -boot-load-size 4 -boot-info-table \
-   -eltorito-alt-boot -e boot/grub/efi.img -no-emul-boot \
-   -o ../"$ISO_NAME.iso" .
-sudo isohybrid -u ../"$ISO_NAME.iso"
+#isohybrid used only in 64bit architecture
```
Comment (bencaradocdavies) [inline bin/build_chroot.sh]: And this comment should be "UEFI supported only in amd64 architecture".

Comment (kalxas) [inline bin/build_chroot.sh]: ok, will fix, thanks

**Comment by kalxas** [comment]
I was able to build alpha1 for i386 using this PR

**Pull request merged**